src/main/kotlin/snc/openchargingnetwork/node/components/OcpiMessageHandler.kt

```diff
@@ -45,7 +45,7 @@ open class OcpiMessageHandler(val request: OcpiRequestVariables,
      *
      */
     fun isSigningActive(recipient: BasicRole? = null): Boolean {
-        var active = properties.signatures || request.headers.signature != null
+        var active = properties.signatures || (request.headers.signature !=null && request.headers.signature != "undefined")
```

Comment:
Would it make just to use "logical not" here? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT
```suggestion
        var active = properties.signatures || !request.headers.signature

Reply:
@jrhender I don't think it is undefined as a value, it is undefined as a string, so that wouldn't register as null or undefined as truly undefined.

Reply:
literally coming back as "undefined" not undefined